packages/nimble-components/src/table/virtualization-helper.ts

```diff
@@ -0,0 +1,129 @@
+import { observable } from '@microsoft/fast-element';
+import {
+    Virtualizer,
+    VirtualizerOptions,
+    elementScroll,
+    observeElementOffset,
+    observeElementRect,
+    VirtualItem
+} from '@tanstack/virtual-core';
+import { controlHeight } from '../theme-provider/design-tokens';
+import type { Table } from '.';
+import type { TableRecord } from './types';
+
+/**
+ * Table virtualization helper
+ */
+export class TableVirtualizationHelper<
+    TData extends TableRecord = TableRecord
+> {
+    /**
+     * @internal
```

Comment:
Could we just mark the entire class `@internal` instead of each public property?

Reply:
Done